views/components/Sidebar/SidebarLink.jsx

```diff
@@ -1,13 +1,15 @@
 import React, { forwardRef } from 'react'
+import classNames from 'classnames'
 import { NavLink } from 'react-router-dom'
+
 import './SidebarLink.css'
 
 const SidebarLink = forwardRef((props, ref) => (
   <NavLink
     ref={ref}
     to={props.to}
     className={({ isActive }) =>
-      isActive ? 'SidebarLink_link-active' : 'SidebarLink_link'
+      classNames('SidebarLink_link', { 'SidebarLink_link-active': isActive })
```

Comment:
I'm confused about this change it doesn't really seem to fix anything that was broken..?
```suggestion
      classNames('SidebarLink_link', { 'SidebarLink_link-active': isActive })
```

Reply:
it goes along with the changes to the CSS. `SidebarLink_link` has all the shared styles and the active modifier class only has the modifier-specific styles.